multiqc/modules/checkqc/checkqc.py

```diff
@@ -472,6 +480,30 @@ def add_undetermined_percentage_section(self):
             plot=bargraph.plot(data, cats, pconfig),
         )
 
+    def add_undetermined_percentage_zero_yield_section(self):
+        """Add a section for lanes with zero yield
+
+        Creates a table with the lane names with zero yield
+
+        Args:
+            run (str): name of sequencing run
+        """
+        data = self.checkqc_data["UndeterminedPercentageHandler_ZeroYield"]
+
+        pconfig = {"id": "checkqc_zero-yield-table", "table_title": "CheckQC: Lanes with Yield 0", "scale": "Reds"}
```

Comment:
I think it would be less confusing to set `col1_header` to something else than the default "Sample Name". Maybe `Lane Number`? It's a bit overkill since the other column also is specifying lane. I guess only a list is needed really. Is it possible to make a table with only the first column? I've only made lists in MultiQC reports by adding a HTML list using the custom content module before.

Reply:
You are right, the table columns are somewhat redundant. I don't think it is possible to only have the first column, because this column is created automatically by MultiQC using the sample names. I used the lane names because I don't have any other useful info to put in the cell.
I don't find documentation about HTML content in a module, I'm not sure if that is easily possible as with custom content. It would be possible to simply add the list of failed lanes to the section text separated by commas, and have no plot/table at all.

Reply:
I would suggest to use "run" as "sample" (line 149) instead of "lane", and `col1_header="Run"`. That also removes the need for the special case of one run. I tried it out:
![zero_yield_table](https://user-images.githubusercontent.com/1584541/143205933-c4bb6f36-edcb-47f7-9637-599706acf65c.JPG)
What do you think?

Reply:
The changes I tested: https://github.com/matrulda/MultiQC/commit/9051fa6b4a271706c34a3cc163c1a01b1914b6f4

Reply:
Thats a good solution! I pulled your commit

Reply:
Nice, thanks :smiley: